benchmark/_test-double-benchmarker.js

```diff
@@ -18,12 +23,21 @@ function request(res) {
       run();
     } else {
       console.log(JSON.stringify({ throughput }));
+      if (client) {
+        client.destroy();
+      }
     }
   });
 }
 
 function run() {
-  http.get(url, request);
+  if (http.get) {
```

Comment:
nit
```suggestion
  if (myModule === 'http') {
```
makes is clear what we're benching